webapp/static/css/toast.css

```
@@ -0,0 +1,149 @@
+  padding: 12px 20px;
+  box-shadow: 0 4px 12px rgba(0, 0, 0, 0.15);
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  min-width: 250px;
+  max-width: min(420px, calc(100vw - 40px));
+  /* **ההיסט יחסי לרוחב הכרטיס, לא קבוע.** ``100%`` בטרנספורם נמדד מול
+   * האלמנט עצמו, ולכן הוא יוצא במלואו בכל רוחב. היסט קבוע של 400px היה
+   * קטן מ-``max-width`` של 420px, והשאיר הודעה ארוכה נראית למחצה. */
+  --ck-toast-exit: calc(100% + 20px);
+  transform: translateX(var(--ck-toast-exit));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)